spec/unit/webrtc/call.spec.ts

```diff
@@ -598,13 +557,14 @@ describe('Call', function() {
                 videoMuted: false,
             }),
         ]);
-        await client.httpBackend.flush();
+        await client.httpBackend.flush("");
         await callPromise;
         call.getOpponentMember = jest.fn().mockReturnValue({ userId: "@bob:bar.uk" });
 
-        call.updateRemoteSDPStreamMetadata({
+        (call as any).updateRemoteSDPStreamMetadata({
             "remote_usermedia_stream_id": {
                 purpose: SDPStreamMetadataPurpose.Usermedia,
+                // XXX: This type doesn't include 'id'?
                 id: "remote_usermedia_feed_id",
```

Comment:
If you could delete that please, it was mistakenly added during some experiments